### DISCUSSION TITLE
Presentation of restrictions

### DISCUSSION THREAD

**Discussion by parfait8566** [Jitendex/Jitendex#74] [Ideas]
For example, the entry "十戒" currently looks like this:

- 1st sense〔十戒 only〕
- 2nd sense

I think it would be immediately clearer if the restriction information was shown more like this:
- 1st sense 〔十戒〕
- 2nd sense〔十戒・十誡〕

I guess it could look awkward if there are a ton of possible forms/readings for the same sense, but wouldn't it be possible to make it that if they exceed a certain number they revert to the previous look?

**Comment by stephenmk** [comment]
Thanks, that's an interesting idea. I'll have to make a list of all the entries with restrictions (I'm sure there are thousands) and look through them to get a feel for how that might look. I can't guarantee that I'll make any changes. My first inclination for things like this is to say that the change should be made in the source database so that I don't have to maintain my own set of customizations. For example, maybe the entry for 十戒 would look better if 十誡 was a hidden form and sense 02 had a "also written as 十誡" note.

I don't really like the idea of hiding the word "only" in the turtle-shell braces by default. I could definitely wrap the word in some HTML so that advanced users can hide it if they're tired of seeing it, though.
